city_scrapers/spiders/alle_port_authority.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+import unicodedata
+from datetime import datetime, timedelta
+
+from dateutil.parser import parse
+from lxml import html
+
+from city_scrapers.constants import BOARD, COMMITTEE
+from city_scrapers.spider import Spider
+
+
+class AllePortAuthoritySpider(Spider):
+    name = 'alle_port_authority'
+    agency_name = 'Port Authority of Allegheny County'
+    timezone = 'America/New_York'
+    allowed_domains = ['www.portauthority.org']
+    start_urls = ['https://www.portauthority.org/paac/CompanyInfoProjects/'
+                  'BoardofDirectors/MeetingAgendasResolutions.aspx']
+    event_year = str(datetime.now().year)
```

Comment:
The `str` call here is unnecessary since you're using it in the `format` function later